### PR TITLE
refactor partitioning to use cluster.KafkaPartitioner

### PR DESCRIPTION
**Pull request by woodsaj**
We now support partitioning byOrg or bySeries.  bySeries is the new default.

### CODE REVIEW

**Comment by woodsaj** [comment]
this is blocked by https://github.com/raintank/metrictank/pull/425

File: main.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ var (
 	publishMetrics   = flag.Bool("publish-metrics", false, "enable metric publishing")
 	eventTopic       = flag.String("event-topic", "events", "NSQ topic for events")
 	publishEvents    = flag.Bool("publish-events", false, "enable event publishing")
+	partitionBy      = flag.String("partition-by", "bySeries", "method used for paritioning metrics. (byOrg|bySeries)")
```
Comment (Dieterbe) [inline main.go]: it's redundant to have the `by` in both the flag as well as the value.
suggested alternatives:
`partition byOrg|bySeries` or `partition-by org|series`

**Comment by woodsaj** [comment]
@Dieterbe  Can this be merged now?

File: scripts/config/tsdb.ini [hunk 2]
```diff
@@ -8,6 +8,7 @@ kafka-comp = none
 publish-metrics = false
 event-topic = events
 publish-events = false
+partition-by = bySeries
```
Comment (Dieterbe) [inline scripts/config/tsdb.ini]: outdated variable name

**Comment by Dieterbe** [comment]
other than my comment, LGTM

**Pull request merged**